integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/context/resolver/Resolver.java

```diff
@@ -0,0 +1,10 @@
+package com.redhat.service.bridge.integration.tests.context.resolver;
+
+import com.redhat.service.bridge.integration.tests.context.TestContext;
+
+public interface Resolver {
+
+    boolean match(String placeholder);
+
+    String replace(String content, TestContext context);
+}
```

Comment:
Could we please document this interface?

Reply:
updated